### ISSUE TITLE
keepWarnings option like in postcss-log-warnings

### ISSUE THREAD

**Issue by gazay** [postcss/postcss-messages#9]
I think by default we better not clear warnings after append them to result because there can be used other plugin after, like postcss-log-warnings. So option should be named as `clearWarnings`

**Comment by MoOx** [comment]
+1.
I plan to integrate messages api in cssnext soon and I will really need to get both methods at the same time :)

**Comment by ai** [comment]
I agree that `clearWarnings` option is better, because it is OK if we will show warning i browsers and CLI too.

**Comment by MoOx** [comment]
+1 for clearWarnings. Maybe @davidclark could do the same for postcss-log-warnings.

**Comment by davidtheclark** [comment]
I'm fine with this. Created in issue in log-warnings to get it done.
